src/panel_gwalker/_gwalker.py

```diff
@@ -137,6 +400,8 @@ def _process_param_change(self, params):
                 params["config"] = {}
```

Comment:
When changing to `viewer` the data `object` should be sent. Even if `server_computatation` is True.

When changing from `server_computation=True` to `server_computation=False` the data `object` should be sent to have something to render.

Currently this does not work.

@philippjfr. Can you fix?

https://github.com/user-attachments/assets/accf727e-9991-459f-8cd2-c94be46d281c